assets/js/partners/contact.js

```diff
@@ -43,6 +44,24 @@ async function submitPartnerContact(event) {
   const lastName = fetchTextInput('last_name');
   const jobTitle = fetchTextInput('job_title');
   const email = fetchTextInput('email').toLowerCase();
+
+  const blockedDomains = [
+    'gmail.com',
+    'hotmail.com',
+    'icloud.com',
+    'live.com',
+    'outlook.com',
+    'sbcglobal.net',
+    'yahoo.com',
+  ];
```

Comment:
Minor: This seems like a constant value, so I wouldn't expect to allocate this array each time in the validation logic. Conventionally I might expect to see this defined toward the top of the file (in the topmost scope) and named to indicate that it's a constant, e.g.

```ts
const BLOCKED_DOMAINS = [
  // ...
];

window.clearCaptchaError = () => {
  // ...
```

Reply:
good catch, I'll move this around